lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/metrics/TDigestHistogram.java

```diff
@@ -46,7 +46,7 @@ public TDigestHistogram(TDigest digest)
     @JsonProperty
     public TDigest getDigest()
     {
-        return digest;
+        return TDigest.copyOf(digest);
```

Comment:
How often is that called?

Reply:
serialization or `mergeWith`, so I think not that often.